stable/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -75,7 +75,7 @@ spec:
           image: "{{ .Values.image.csiSnapshotter.repository }}:{{ .Values.image.csiSnapshotter.tag }}"
           args:
             - --csi-address=$(ADDRESS)
-            - "-leader-election"
+            - -leader-election
```

Comment:
doesn't it need to be double-dash ?

Reply:
look at https://github.com/kubernetes-sigs/azuredisk-csi-driver for the next upgrade

Reply:
I will ignore the inconsistency of the also in upstream existing arguments 🤣 ... for now 😉